RestComm/NUnitTestClass.cs

```diff
@@ -0,0 +1,16 @@
+﻿using NUnit.Framework;
+using System;
+
+namespace RestComm
+{
+	[TestFixture ()]
```

Comment:
I was using Monodevelop with Nunit version 2.6.4(this is latest version of nunit v2) . Can you try once in monodevelop and tell me if you are still getting same error .This problem might be specific to visual studio only .